libs/modelviewer/src/providers/selectionProvider.tsx

```diff
@@ -95,19 +95,22 @@ export const SelectionContextProvider = ({
     }
   }, [tagsOverlay]);
```

Comment:
This useEffect also seems unnecessary. We can pair program on this if you want.

Reply:
Fixed